test/check-cli

```diff
@@ -104,7 +104,7 @@ class TestTar(composertest.ComposerTestCase):
 
 class TestPylint(composertest.ComposerTestCase):
     def test_ci(self):
-        self.runCliTest("/tests/cli/test_ci.sh")
+        self.runImageTest("/tests/cli/test_ci.sh")
```

Comment:
The 3rd commit must be squashed with the first one.

Reply:
I believe this is solved now.